documentation/authentication.md

```diff
@@ -96,7 +96,7 @@ spec:
       secretName: apikey
   containers:
   - name: dotnetmonitoragent
-    image: mcr.microsoft.com/dotnet/dotnet-monitor:6.0.0-preview.8
+    image: mcr.microsoft.com/dotnet/dotnet-monitor
```

Comment:
A lot of AKS docs discourage using latest tags in production. IIRC we also decided that breaking changes are ok for major version boundaries so we probably would want customers to be explicit about their major version tag.

Reply:
For AKS, I'll add back the tag, but only as `6.0` to discourage pinning to patch releases.